### DISCUSSION TITLE
Provide a way to disable ens name & avatar requests.

### DISCUSSION THREAD

**Discussion by songkeys** [rainbow-me/rainbowkit#593] [Ideas]
Please see https://github.com/rainbow-me/rainbowkit/discussions/429

**Comment by markdalgleish** [comment]
As of [RainbowKit v0.4.1,](https://github.com/rainbow-me/rainbowkit/releases/tag/%40rainbow-me%2Frainbowkit%400.4.1) we only ever request ENS names and avatars from mainnet, and in [RainbowKit v0.4.4](https://github.com/rainbow-me/rainbowkit/releases/tag/%40rainbow-me%2Frainbowkit%400.4.4) we now only make ENS requests when mainnet is in your list of configured chains. Does this address your issue?

**Reply by songkeys** [reply]
Yes. Thank you for this feature!
